tests/virtctl/create_vm.go

```diff
@@ -482,48 +500,41 @@ func setFlag(flag, parameter string) string {
 	return fmt.Sprintf("--%s=%s", flag, parameter)
 }
 
-func unmarshalVM(bytes []byte) *v1.VirtualMachine {
-	vm := &v1.VirtualMachine{}
-	Expect(yaml.Unmarshal(bytes, vm)).To(Succeed())
-	Expect(vm.Kind).To(Equal("VirtualMachine"))
-	Expect(vm.APIVersion).To(Equal("kubevirt.io/v1"))
-	return vm
+func runCmd(args ...string) ([]byte, error) {
+	_args := append([]string{create.CREATE, VM}, args...)
+	return clientcmd.NewRepeatableVirtctlCommandWithOut(_args...)()
 }
 
-func createInstancetype(virtClient kubecli.KubevirtClient) *instancetypev1beta1.VirtualMachineInstancetype {
-	instancetype := &instancetypev1beta1.VirtualMachineInstancetype{
-		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: "vm-instancetype-",
-			Namespace:    util.NamespaceTestDefault,
-		},
-		Spec: instancetypev1beta1.VirtualMachineInstancetypeSpec{
-			CPU: instancetypev1beta1.CPUInstancetype{
-				Guest: uint32(1),
-			},
-			Memory: instancetypev1beta1.MemoryInstancetype{
-				Guest: resource.MustParse(size),
-			},
-		},
+func decodeVM(bytes []byte) (*v1.VirtualMachine, error) {
```

Comment:
would it be possible to reuse the same fn which is in pkg/virtctl/create/vm/vm_test.go?